src/DotNetDoodle.Analyzers/DotNetDoodle.Analyzers/VarDiagnosticAnalyzer.cs

```diff
@@ -38,13 +38,13 @@ public override void Initialize(AnalysisContext context)
         private void AnalyzeNode(SyntaxNodeAnalysisContext context)
         {
             VariableDeclarationSyntax variableDeclaration = (VariableDeclarationSyntax)context.Node;
-            if (IsLegitVarUsage(variableDeclaration, context.SemanticModel))
+            if (IsIllegitVarUsage(variableDeclaration, context.SemanticModel))
             {
                 context.ReportDiagnostic(Diagnostic.Create(Rule, variableDeclaration.Type.GetLocation()));
             }
         }
 
-        private bool IsLegitVarUsage(VariableDeclarationSyntax variableDeclaration, SemanticModel semanticModel)
+        private bool IsIllegitVarUsage(VariableDeclarationSyntax variableDeclaration, SemanticModel semanticModel)
```

Comment:
I would prefer something positive here, like `IsReportable`. This makes it hard to read and understand the cases :s